chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
+#include<iostream>
+
+int Chess2::perform(str Move) {
+    int original;
+    original = board[Move[2] - 48][Move[3] - 48];
```

Comment:
Move[3] - 48 this is wrong. There are multiple ways we can do it, but at least 48 should be in constant.